### PR TITLE
correcting typos in readme file

### PR DESCRIPTION
**Pull request by ghostinthewires**
Signed-off-by: Andrew Urwin <ellieniwru@hotmail.co.uk>

## Description
Correcting several typos and grammar issues in readme.md file

## Motivation and Context
Removes several typos and errors from readme file of faas-cli

## How Has This Been Tested?
Documentation - no testing but requires verification

## Types of changes
Documentation

## Checklist:

- [X] My code follows the code style of this project.
- [X] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.
- [X] I've read the [CONTRIBUTION](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md) guide
- [X] I have signed-off my commits with `git commit -s`
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,13 +22,13 @@ You can install the CLI with a `curl` utility script, `brew` or by downloading t
 Utility script with `curl`:
 
 ```
-$ curl -sSL https://cli.openfaas.com | sudo sh
+$ curl -ssl https://cli.openfaas.com | sudo sh
```
Comment (alexellis) [inline README.md]: This is not a typo, please revert it.

Comment (alexellis) [reply README.md]: Use `git commit --amend -s`

File: README.md [hunk 2]
```diff
@@ -69,7 +69,7 @@ The main commands supported by the CLI are:
 * `faas-cli store` - allows browsing and deploying OpenFaaS store functions
 * `faas-cli secret` - manage secrets for your functions
 
-The default gateway URL of `127.0.0.1:8080` can be overriden in three places including an environmental variable.
+The default gateway URL of `127.0.0.1:8080` can be overridden in three places including an environmental variable.
```
Comment (alexellis) [inline README.md]: 👍

File: README.md [hunk 3]
```diff
@@ -90,7 +90,7 @@ The default template URL of `https://github.com/openfaas/templates.git` can be o
 * 1st priority CLI input
 * 2nd priority `OPENFAAS_TEMPLATE_URL` environmental variable
 
-Help for all of the commands supported by the CLI can be found by running:
+Help for all the commands supported by the CLI can be found by running:
```
Comment (alexellis) [inline README.md]: This one seems a little ambiguous, perhaps a little bit too prescriptive? https://jakubmarian.com/is-it-all-the-or-all-of-the-in-english/

File: README.md [hunk 4]
```diff
@@ -104,7 +104,7 @@ In your YAML you can also specify `lang: node/python/go/csharp/ruby`
 
 * Supports common languages
 * Quick and easy - just write one file
-* Specify depenencies on Gemfile / requirements.txt or package.json etc
+* Specify dependencies on Gemfile / requirements.txt or package.json etc
```
Comment (alexellis) [inline README.md]: 👍

File: README.md [hunk 5]
```diff
@@ -177,7 +177,7 @@ Read the [YAML reference guide in the OpenFaaS docs](https://docs.openfaas.com/r
 
 A YAML stack file groups functions together and also saves on typing.
 
-You can define individual functions or a set of of them within a YAML file. This makes the CLI easier to use and means you can use this file to deploy to your OpenFaaS instance.  By default the faas-cli will attempt to load `stack.yaml` from the current directory.
+You can define individual functions or a set of them within a YAML file. This makes the CLI easier to use and means you can use this file to deploy to your OpenFaaS instance.  By default the faas-cli will attempt to load `stack.yaml` from the current directory.
```
Comment (alexellis) [inline README.md]: 👍

**Comment by ghostinthewires** [comment]
I seem to be having issues with this Alex. Even though the commit shows as signed off at this end, whenever I push it, Derek shows as not signed. Sorry for being a pain :(

**Comment by alexellis** [comment]
Don't worry, it can be fixed. One of your commits will be missing the sign off message. I don't think you need for commits for this so why don't you just reset and then do a single commit?

**Comment by alexellis** [comment]
@rgee0 please merge when ready

**Pull request merged**